### PR TITLE
fix(manager-components): pci-header fix k8s header urls

### PR DESCRIPTION
**Pull request by seven-amid**
| Question         | Answer
| ---------------- | ---
| Branch?          | feat/pci-kubernetes
| Bug fix?         | yes
| New feature?     | no
| Breaking change? | no
| Tickets          | DTCORE-2530
| License          | BSD 3-Clause

- [x] Try to keep pull requests small so they can be easily reviewed.
- [x] Commits are signed-off
- [x] Only FR translations have been updated
- [x] Branch is up-to-date with target branch
- [x] Lint has passed locally
- [ ] Standalone app was ran and tested locally
- [x] Ticket reference is mentioned in linked commits (internal only)
- [ ] Breaking change is mentioned in relevant commits

### CODE REVIEW

**Review by SimonChaumet — APPROVED** [review]

**Review by anooparveti — CHANGES_REQUESTED** [review]
Change on `manager-components` must be in an independent PR and tracked by a ticket on Github project board https://github.com/orgs/ovh/projects/22

File: packages/manager-components/src/components/guides-header/guides-header.component.tsx [hunk 1]
```diff
@@ -60,7 +60,9 @@ export function GuidesHeader({
         <GuidesHeaderItem
           key={guide}
           guide={guides[guide]}
-          href={`${guides[guide].url[ovhSubsidiary]}`}
+          href={
+            guides[guide].url[ovhSubsidiary] || guides[guide].url["DEFAULT"]
```
Comment (chipp972) [inline packages/manager-components/src/components/guides-header/guides-header.component.tsx:64]:
```suggestion
            guides[guide].url[ovhSubsidiary || 'DEFAULT']
```

Comment (seven-amid) [reply packages/manager-components/src/components/guides-header/guides-header.component.tsx:64]: What you proposed doesn't fix my problem. If we suppose that the subsidiary exists (like FR, for example), but there is no FR entry in my guidesUrl object:

```
{
  GB: 'url-gb',
  IE: 'url-ie',
  DEFAULT: 'default-url',
  US: 'url-us',
}
```

In this case, I want to use the default URL. With your proposal, `guides[guide].url[ovhSubsidiary || 'DEFAULT']`, the output would be `undefined`.

**Review by aboungnaseng-ovhcloud — APPROVED** [review]

File: packages/manager-components/src/components/guides-header/guides-header.component.tsx (hunk 1 above)
Comment (anooparveti) [inline packages/manager-components/src/components/guides-header/guides-header.component.tsx:64]: > Check failure on line 64 in packages/manager-components/src/components/guides-header/guides-header.component.tsx

GitHub Actions
/ Lint Code Base

["DEFAULT"] is better written in dot notation

Can you please fix this lint error?
```suggestion
            guides[guide].url[ovhSubsidiary] || guides[guide].url.DEFAULT
```

Comment (anooparveti) [inline packages/manager-components/src/components/guides-header/guides-header.component.tsx:64]: > Check failure on line 64 in packages/manager-components/src/components/guides-header/guides-header.component.tsx

GitHub Actions
/ Lint Code Base

["DEFAULT"] is better written in dot notation

Can you please fix this lint error?
```suggestion
            guides[guide].url[ovhSubsidiary] || guides[guide].url.DEFAULT
```

**Review by anooparveti — CHANGES_REQUESTED** [review]

**Pull request merged**